### PR TITLE
Update Workflow to push image on Docker Hub

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
[Update Workflow to push image on Docker Hub](https://github.com/Exodus-Privacy/exodus-standalone/issues/26)
Add two jobs on workflow:

- First job to create `latest` image after each push on master branch
- Second job to create `tag` image after each tag push

`build_latest_image` and `build_tag_image` is execute only if job `build` is successful.

Just needs add token and username docker on secrets

### CODE REVIEW

**Comment by Jean-BaptisteC** [comment]
@pnu-s can you check actions permissions on settings to authorize Docker Actions

**Comment by pnu-s** [comment]
This way, it looks like we are doing some steps twice where we could do them only once.

Could we have only one `push` job, with the `if` conditions in the end?

**Comment by pnu-s** [comment]
I'll take a look on how to add the required token.

**Comment by Jean-BaptisteC** [comment]
To allow Docker actions, you need to changes policies in organization settings -> actions permissions -> policies

**Comment by Jean-BaptisteC** [comment]
[How create docker Token](https://docs.docker.com/docker-hub/access-tokens/)

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -36,3 +36,29 @@ jobs:
       - uses: hadolint/hadolint-action@v1.5.0
         with:
           dockerfile: Dockerfile
+  build_image:
+    needs: build
+    if: github.event_name == 'push'
```
Comment (pnu-s) [inline .github/workflows/main.yml:41]: Can we make sure it is on the `master` branch?

**Comment by pnu-s** [comment]
Seems like the checks did not run, do you know why?

Otherwise, LGTM

**Review by pnu-s — APPROVED** [review]

**Comment by Jean-BaptisteC** [comment]
Check this to allow workflow Docker https://github.com/Exodus-Privacy/exodus-standalone/pull/42#issuecomment-1086900392
Check this to create Github Secrets https://github.com/Exodus-Privacy/exodus-standalone/pull/42#issuecomment-1121437179

**Comment by pnu-s** [comment]
Secrets are already created.
All workflows should now be allowed.

File: .github/workflows/main.yml (hunk 1 above)
Comment (Jean-BaptisteC) [reply .github/workflows/main.yml:41]: all jobs use global settings set in the top of the file

**Comment by Jean-BaptisteC** [comment]
I have update runs-on to trigger workflow

**Pull request merged**